modules/service-ca-certificates.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="service-ca-certificates_{context}"]
+= Service CA certificates
+
+== Purpose
+
+`service-ca` is an Operator that creates a self-signed CA, which is valid for one year,
```

Comment:
Valid for 14 months as of 4.4 (and is the process of being backported to 4.3, 4.2, and 4.1)